hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/ddl/JDBCExecutor.java

```diff
@@ -65,7 +65,7 @@ public void runSQL(String s) {
     try {
       stmt = connection.createStatement();
       LOG.info("Executing SQL " + s);
-      stmt.execute(s);
+      stmt.execute(escapeAntiSlash(s));
```

Comment:
Is the escape related with this change ?

Reply:
yes, both JDBC and HMS generate sql and antislash should be doubled escaped otherwise it is lost.

Antislash is used to escape double quotes in the comments DDL.